channels/dk.m3u

```diff
@@ -1,23 +1,23 @@
+#EXTINF:-1 tvg-id="KKRtv.dk" tvg-name="KKRtv" tvg-country="DK" tvg-language="Danish" tvg-logo="https://i.imgur.com/cpl90ZC.png" group-title="",KKRtv (720p)
 http://stream.kkr.dk/live/kkr/chunklist_w832247479.m3u8
 #EXTINF:-1 tvg-id="KanalHovedstaden.dk" tvg-name="Kanal Hovedstaden" tvg-country="DK" tvg-language="Danish" tvg-logo="https://i.imgur.com/hATNB38.jpg" group-title="",Kanal Hovedstaden
 https://59b954022ec35.streamlock.net/liveTV2/_definst_/smil:liveTVstream2.transcoder.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="KKRTV.dk" tvg-name="KKR TV" tvg-country="DK" tvg-language="" tvg-logo="" group-title="",KKR TV (720p)
+#EXTINF:-1 tvg-id="KKRtv.dk" tvg-name="KKRtv" tvg-country="DK" tvg-language="Danish" tvg-logo="https://i.imgur.com/cpl90ZC.png" group-title="",KKRtv (720p)
 http://stream.kkr.dk/live/kkr/playlist.m3u8
-#EXTINF:-1 tvg-id="KKRTV.dk" tvg-name="KKR TV" tvg-country="DK" tvg-language="" tvg-logo="http://kkrtv.dk/wp-content/uploads/2015/09/kkrtv-logo-navi.png" group-title="",KKR TV (720p)
+#EXTINF:-1 tvg-id="KKRtv.dk" tvg-name="KKRtv" tvg-country="DK" tvg-language="Danish" tvg-logo="https://i.imgur.com/cpl90ZC.png" group-title="",KKRtv (720p)
 http://stream.kkr.dk/live/ngrp:kkr_adaptive/chunklist.m3u8
 #EXTINF:-1 tvg-id="KKRtv.dk" tvg-name="KKRtv" tvg-country="DK" tvg-language="Danish" tvg-logo="https://i.imgur.com/cpl90ZC.png" group-title="",KKRtv (720p)
 http://stream.kkr.dk/live/ngrp:kkr_adaptive/playlist.m3u8
-#EXTINF:-1 tvg-id="TV 2 Bornholm DK" tvg-name="TV 2 Bornholm DK" tvg-country="DK" tvg-language="Danish" tvg-logo="https://i.imgur.com/ylB5ea2.jpg" group-title="",TV 2/Bornholm (1080p)
+#EXTINF:-1 tvg-id="TV2Bornholm.dk" tvg-name="TV2/Bornholm" tvg-country="DK" tvg-language="Danish" tvg-logo="https://i.imgur.com/ylB5ea2.jpg" group-title="",TV2/Bornholm (1080p)
 https://live.tv2bornholm.dk/stream/live/chunklist.m3u8
-#EXTINF:-1 tvg-id="TV 2 Ostjylland DK" tvg-name="TV 2 Ostjylland DK" tvg-country="DK" tvg-language="Danish" tvg-logo="https://i.imgur.com/rC07Op2.jpg" group-title="",TV 2/Østjylland (720p) [Offline]
+#EXTINF:-1 tvg-id="TV2Ostjylland.dk" tvg-name="TV2/Østjylland" tvg-country="DK" tvg-language="Danish" tvg-logo="https://i.imgur.com/rC07Op2.jpg" group-title="",TV2/Østjylland (720p) [Offline]
 https://cdn-lt-live.tv2oj.dk/dc-1/live/hls/p/2102081/e/0_x4p3licd/sd/6000/t/5aoqiGIZk5dVwQnsYa1OiQ/master.m3u8
-#EXTINF:-1 tvg-id="TV2Fyn.dk" tvg-name="TV2 Fyn" tvg-country="DK" tvg-language="" tvg-logo="https://vignette1.wikia.nocookie.net/logopedia/images/6/6e/TV_2_Fyn.png" group-title="",TV2 Fyn (360p)
+#EXTINF:-1 tvg-id="TV2Fyn.dk" tvg-name="TV2/Fyn" tvg-country="DK" tvg-language="" tvg-logo="https://vignette1.wikia.nocookie.net/logopedia/images/6/6e/TV_2_Fyn.png" group-title="",TV2 Fyn (360p)
 http://cdn-lt-hls-vod.tv2fyn.dk/fhls/p/1966291/sp/196629100/serveFlavor/entryId/0_yct7rqn3/v/2/flavorId/0_h2wfmuqm/name/a.mp4/index.m3u8
```

Comment:
Alternative could be https://cdnapisec.kaltura.com/p/1966291/sp/1966291/playManifest/entryId/0_yct7rqn3/format/applehttp/protocol/https/flavorId/0_h2wfmuqm/a.m3u8